runtime_scan/pkg/orchestrator/scanconfigwatcher/schedule.go

```diff
@@ -167,10 +196,17 @@ func isCronPeriodic(c *cronexpr.Expression) bool {
 type ScheduleState int8
 
 const (
+	// ScheduleStateDisabled means the ScanConfig isi disabled no Scan needs to be scheduled for it.
 	ScheduleStateDisabled ScheduleState = iota
+	// ScheduleStateUnscheduled means the ScanConfig does not define a schedule.
 	ScheduleStateUnscheduled
+	// ScheduleStateNotDue means the OperationTime for ScanConfig is in the future based on the current ScheduleWindow.
 	ScheduleStateNotDue
+	// ScheduleStateDue means the OperationTime for ScanConfig is in the current ScheduleWindow.
 	ScheduleStateDue
+	// ScheduleStateOverdue means the OperationTime for ScanConfig is in the past compared to
+	// the current ScheduleWindow, but it has recurring schedule, therefore the new OperationTime needs
+	// to be calculated which is in the next ScheduleWindow.
```

Comment:
```suggestion
	// the current ScheduleWindow, but it has recurring schedule, therefore the new OperationTime needs
	// to be calculated from the cron expression.
```

I'm not sure "in the next ScheduleWindow" in the right wording here, I've provided a suggestion.

Reply:
Fixed.